pkg/operator/workloadpartitioning/cpuPartitioningAlertRenderer.go

```diff
@@ -0,0 +1,113 @@
+package workloadpartitioning
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"text/template"
+
+	v1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-kube-apiserver-operator/bindata"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceread"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/utils/cpuset"
+)
+
+const defaultCoresNum = 4
```

Comment:
Is it really safe to assume that all the SNO clusters that are running without CPU partitioning will have 4 cores? If we ever have a case where the machine has 2,6,8,.. cores, the alerts will become incorrect.

Maybe we could use `runtime.NumCPU()` from the go standard library to get an accurate value for the number of CPU available?